barretenberg/cpp/src/barretenberg/flavor/flavor.hpp

```diff
@@ -85,9 +85,9 @@ namespace bb {
  */
 class PrecomputedEntitiesBase {
   public:
-    size_t circuit_size;
-    size_t log_circuit_size;
-    size_t num_public_inputs;
+    uint64_t circuit_size;
```

Comment:
had to change these to serializable types (uint64_t), but this led to many compile errors relating to implicit conversions losing precision, so I had to add many static_casts as a temporary fix. The long term fix should be using uint32_t/uint64_t instead of size_t by default in most places that aren't accessing memory